authentication/templates/registration/register.html

```diff
@@ -0,0 +1,39 @@
+{% extends "registration/base.html" %}
+
+{% block title %}
+<title> Register </title>
+{% endblock %}
+
+{% block content %}
+
+<h5 class="text-center"> Register </h5>
+<form method="POST" action="/accounts/register/">
+    {% csrf_token %}
+    {% for message in messages %}
+    <h5 class="text-center text-danger"> {{message}} </h5>
+    {% endfor %}
+    <div class="mb-3">
+        <label for="username" class="form-label">Username</label>
+        <input type="text" class="form-control" id="username" name="username">
+    </div>
+    <div class="mb-3">
+        <label for="first_name" class="form-label">First Name</label>
+        <input type="text" class="form-control" id="first_name" name="first_name">
+    </div>
+    <div class="mb-3">
+        <label for="last_name" class="form-label">Last Name</label>
+        <input type="text" class="form-control" id="last_name" name="last_name">
+    </div>
+    <div class="mb-3">
+        <label for="email" class="form-label">Email</label>
```

Comment:
It's great that you implemented your own form, but you need to add field validation as well. I could register without even entering an email and password in the current case.

Btw you can also use these for the same purpose:
`from django.contrib.auth.views import LoginView` for login purpose
`from django.contrib.auth.forms import UserCreationForm` for registration.